lib/modules/manager/cnb/readme.md

```diff
@@ -0,0 +1,25 @@
+The `cnb` manager updates Cloud Native Buildpacks project descriptors in `project.toml` files.
+A `project.toml` file can reference builder / buildpack images by URIs.
+Renovate can update a `project.toml` file if:
+
+- It can find the file
+- The file follows the [project descriptor specifications](https://github.com/buildpacks/spec/blob/main/extensions/project-descriptor.md)
```

Comment:
You commited [ignore unsupported buildpack sources](https://github.com/renovatebot/renovate/pull/30799/commits/2b174a9b6933f2345809836b9650f35bfb15b21a).

Do we need to update the readme, so it explains the unsupported buildpack sources?

Reply:
done